practice-typescript/src/scripts/views/user.ts

```diff
@@ -93,6 +92,7 @@ class UserView {
         if (!isUserExist(user.email)) {
           toastMessage(this.toast, MESSAGE_ERROR.ACCOUNT_EXIST, 'toast__error');
         } else {
```

Comment:
This function for handling submitting events is complex. Please check and optimize for this function

Reply:
fixed and I can only delete one else statement